e2e/test/scenarios/question/notebook-data-source.cy.spec.ts

```diff
@@ -469,6 +472,82 @@ describe("issue 28106", () => {
   );
 });
 
+describe("issue 32252", () => {
+  beforeEach(() => {
+    restore("setup");
+    cy.signInAsAdmin();
+
+    createCollection({ name: "My collection" }).then(({ body: collection }) => {
+      createQuestion({
+        name: "My question",
+        collection_id: collection.id,
+        query: {
+          "source-table": ORDERS_ID,
+        },
+      });
+    });
+  });
+
+  it("refreshes data picker sources after archiving a collection (metabase#32252)", () => {
```

Comment:
This test uses the most recent reproduction steps (archiving a collection with questions)